addons/crm/static/tests/tours/crm_email_and_phone_propagation.js

```diff
@@ -49,6 +49,8 @@ odoo.define('crm.crm_email_and_phone_propagation', function (require) {
             extra_trigger: '.o_form_sheet_bg .fa-exclamation-triangle:not(.o_invisible_modifier)',
             content: 'Save the lead and exit to kanban',
             run: 'click',
-        },
+        },{
+            trigger: '.o_kanban_renderer',
+        }
```

Comment:
I did this commit to fix the tour. But anyway, dbo was first: https://github.com/odoo-dev/odoo/commit/fbc86684a8a4e45afde70a7e7c772b41a1b86ad7

Reply:
my bad, since i said i was doing it, i finished the job - didn't know you noticed the issue in between :p